### ISSUE TITLE
Pipe's default timeout overwrites default in cypress.json

### ISSUE THREAD

**Issue by westhomas** [NicholasBoll/cypress-pipe#21]
Pipe's default timeout of 4000 overwrites what's found in cypress.json when no timeout is passed into options.

I think the fix is to use Cypress's timeout as the default:

```
// index.js
  Cypress._.defaults(options, {
    log: true,
    timeout: Cypress.config("defaultCommandTimeout"),
  })
```

See the issue here:

https://github.com/westhomas/cypress-playground/blob/pipe-timeout-issue/cypress/integration/test.spec.js

```
# cypress.json
{
    "defaultCommandTimeout": 30000
}
```

```
// test.spec.js
describe("pipe stuff", function () {

    const doTestpage = () => {
        cy.visit("https://google.com")


        cy.window().then((win) => {
            setTimeout(() => {
                win.document.write("<div id='hi'>I'm here!</div>")
            }, 6000)
        })

    }

    it("should find the div", function () {

        doTestpage()

        const getDiv = win => win.document.getElementById("hi")

        cy
        .window()
        .pipe(getDiv, { timeout: 7000 })
        .should("have.text", "I'm here!")

    })

    it("should timeout too soon", function () {

        doTestpage()


        const getDiv = win => win.document.getElementById("hi")

        cy
        .window()
        .pipe(getDiv)
        .should("have.text", "I'm here!")


    })

})
```

**Comment by NicholasBoll** [comment]
Hey, thanks for pointing that out! You're welcome to open a pull request. If not, I can get that fixed!

**Comment by NicholasBoll** [comment]
Fixed in 1.6.3

**Comment by NicholasBoll** [comment]
Actually fixed in 1.6.4
